packages/cozy-sharing/src/helpers/email.js

```diff
@@ -0,0 +1,11 @@
+// Extract emails address from a string
+export const extractEmails = str => {
+  if (typeof str === 'string')
+    return str.match(/([a-zA-Z0-9._-]+@[a-zA-Z0-9._-]+\.[a-zA-Z0-9_-]+)/gi)
+  return null
+}
+// Validate if a string is an email address or not
+export const validateEmail = str => {
+  const reg = /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>()[\].,;:\s@"]+\.)+[^<>()[\].,;:\s@"]{2,})$/i
```

Comment:
Can we have a short description of what this baby is looking for? Not gonna lie, it's a bit hard to decipher 😁